src/nile/cli.py

```diff
@@ -90,9 +90,11 @@ def run(path, network):
 @click.option("--alias")
 @click.option("--abi")
 @mainnet_token_option
-def deploy(artifact, arguments, network, alias, abi=None, token=None):
+async def deploy(artifact, arguments, network, alias, abi=None, token=None):
```

Comment:
```suggestion
async def deploy(artifact, arguments, network, alias, abi, token):
```

We can remove default None values in click commands arguments because this is the default behavior.